config/initializers/prometheus_metrics.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+if Rails.env.production?
```

Comment:
**not a blocker**  this could run locally if rails server startup included a `bundle exec prometheus_exporter` somehow - could use MakeFile or `config.ru` for example

Reply:
naming wise  I'd call this file just `prometheus.rb`